src/v2/Apps/Conversation/Components/MakeOfferOnInquiryButton.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+import { tappedMakeOffer } from "@artsy/cohesion"
+import styled from "styled-components"
+import { MakeOfferOnInquiryButton_conversation } from "v2/__generated__/MakeOfferOnInquiryButton_conversation.graphql"
+import {
+  CheckCircleIcon,
+  FLAT_SHADOW,
+  Flex,
+  Separator,
+  Text,
+  Button,
+} from "@artsy/palette"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { ConfirmArtworkButtonFragmentContainer } from "./ConfirmArtworkButton"
+
+const ShadowSeparator = styled(Separator)`
+  box-shadow: ${FLAT_SHADOW};
+  width: 100%;
+  height: 0;
+`
+
+export interface MakeOfferOnInquiryButtonProps {
+  openInquiryModal: () => void
+  conversation: MakeOfferOnInquiryButton_conversation
+}
+
+export const MakeOfferOnInquiryButton: React.FC<MakeOfferOnInquiryButtonProps> = ({
+  openInquiryModal,
+  conversation,
+}) => {
+  const tracking = useTracking()
+
+  const liveArtwork = conversation?.items?.[0]?.liveArtwork
+  const artwork = liveArtwork?.__typename === "Artwork" ? liveArtwork : null
+  if (!artwork) return null
+
+  const conversationID = conversation.internalID!
+  const handleOpenModal = () => {
+    tracking.trackEvent(tappedMakeOffer(conversationID ?? ""))
+    openInquiryModal()
+  }
+
+  return (
+    <>
+      <ShadowSeparator />
+      <Flex flexDirection="column" p={1}>
+        <Flex flexDirection="row">
+          <CheckCircleIcon mr={1} />
+          <Flex flexShrink={1}>
+            <Text color="black60" variant="xs" mb={1}>
+              Only purchases completed with our secure checkout are protected by{" "}
+              <RouterLink to="/buyer-guarantee" target="_blank">
+                The Artsy Guarantee
+              </RouterLink>
+              .
+            </Text>
+          </Flex>
+        </Flex>
+        {artwork.editionSets && artwork.editionSets.length > 1 ? (
```

Comment:
Working on [NX-3052](https://artsyproduct.atlassian.net/browse/NX-3052) I saw we use `artwork.isEdition`. Would that be also the case here?